### DISCUSSION TITLE
List of Screens might needed

### DISCUSSION THREAD

**Discussion by M1ngY** [CMPUT301F23T13/DebuggingDemons#26] [Announcements]
I. Item Details Screen:
Add an item (US 01.01.01)
View an item and its details (US 01.02.01)
Edit the details of an item (US 01.03.01)
Delete an item (US 01.04.01)
Attach photographs to an item (US 04.01.01 & US 04.02.01)
Delete an attached photograph (US 04.03.01)
Scan to specify serial number (US 05.01.01)
Scan barcode for product description (US 05.02.01)

II. List of Items Screen:
See a list of items (US 02.01.01)
View total estimated value of items (US 02.02.01)
Select and delete multiple items (US 02.03.01)
Sort items (US 02.04.01)
Filter items by date range (US 02.05.01)
Filter items by description keywords (US 02.06.01)
Filter items by make (US 02.07.01)
Sort and filter items by tags (US 03.04.01 & US 03.05.01)

III. Tags Management Screen:
Define tags (US 03.01.01)
Specify tags for an item (US 03.02.01)
Apply tags to multiple items (US 03.03.01)

IV. Camera & Scanning Screen:
Take photos (US 04.01.01)
Scan for serial number (US 05.01.01)
Scan for product barcode (US 05.02.01)

V. Photo Gallery Selection Screen:
Select photos from the gallery (US 04.02.01)

VI. User Profile Screen:
Create and view a profile with a unique username (US 06.01.01)

**Comment by Ayjc** [comment]
[Storyboard Project Draft.docx](https://github.com/CMPUT301F23T13/DebuggingDemons/files/12885807/Storyboard.Project.Draft.docx)

**Comment by Iris-Ling** [comment]
[List.docx](https://github.com/CMPUT301F23T13/DebuggingDemons/files/12885935/List.docx)

**Comment by elialex15** [comment]
[screensNeeded.docx](https://github.com/CMPUT301F23T13/DebuggingDemons/files/12887537/screensNeeded.docx)
![screensNeeded](https://github.com/CMPUT301F23T13/DebuggingDemons/assets/112982807/79b86472-27c1-4274-94e9-4bfbc1746884)
